recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.7.0" %}
+{% set version = "0.8.0" %}
 
 package:
   name: fairlearn
   version: {{ version }}
 
 source:
   url: https://github.com/fairlearn/fairlearn/archive/v{{ version }}.tar.gz
-  sha256: db5776618f5eadd52d864fa9789473d8b707649b48f3fb27a2445fdf2af093f2
+  sha256: f0e1f96962c4e76352047a095bed4301ae8791eac2d9310f325b52c1ed33a35d
```

Comment:
According to:
https://github.com/fairlearn/fairlearn/releases/tag/v0.8.0
isn't the tag `0c39...` ?

Reply:
this is not the hash of the commit, this is the sha256 of the downloaded archive. You might then change the version to v0.8.0 above?

Reply:
Fair enough.

Reply:
Are you happy with the release? Or are you still working on the release branch?

Reply:
Per my comments on Discord, my only concern is dealing with the docs. I'm not seeing any issues raised by the release itself.

Reply:
Then I'd wait until all is fixed and then push this release out.

Reply:
@riedgar-ms approved on 10/31, does that mean it's ready to merge? Sorry, I'm not 100% aware of what's missing in the docs.